unified_planning/model/problem.py

```diff
@@ -558,20 +569,67 @@ def add_goal(
         if goal_exp != self._env.expression_manager.TRUE():
             self._goals.append(goal_exp)
 
+    def add_trajectory_constraint(self, constraint: "up.model.fnode.FNode"):
+        """
+        Adds the given `trajectory_constraint` to the `Problem`;
+        a trajectory_constraint is an expression defined as:
+            Always, Sometime, At-Most-Once, Sometime-Before, Sometime-After or
+            defined with quantifiers.
+
+        :param trajectory_constraint: The expression added to the `Problem`.
+        """
+        new_traj_list = []
+        simplifier = up.model.walkers.simplifier.Simplifier(self._env, self)
+        (constraint_exp,) = self._env.expression_manager.auto_promote(constraint)
+        new_traj_list.append(
+            simplifier.simplify(
+                self._env.expression_manager.And(
+                    self._trajectory_constraints,
+                    constraint_exp,
+                )
+            )
+        )
+        self._trajectory_constraints = new_traj_list
+
+    def remove_trajectory_constraint(self, constraint: "up.model.fnode.FNode"):
```

Comment:
We don't offer the capability to remove a single goal, action etc. Is this a common use-case or we can just remove this and leave the `clear_trajectory_constraints` method?